deltachat-ios/Controller/SettingsController.swift

```diff
@@ -90,222 +256,90 @@ internal final class SettingsViewController: QuickTableViewController {
-        if !documents.isEmpty {
-            self.hudHandler.showHud(String.localized("one_moment"))
-            DispatchQueue.main.async {
-                self.dcContext.imex(what: what, directory: documents[0])
-            }
-        } else {
-            logger.error("document directory not found")
-        }
+    override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return sections[section].footerTitle
     }
 
-    private func createBackup(_: Row) {
+    // MARK: - actions
+    
+    private func createBackup() {
         let alert = UIAlertController(title: String.localized("pref_backup_export_explain"), message: nil, preferredStyle: .safeActionSheet)
         alert.addAction(UIAlertAction(title: String.localized("pref_backup_export_start_button"), style: .default, handler: { _ in
             self.dismiss(animated: true, completion: nil)
-            self.startImex(what: DC_IMEX_EXPORT_BACKUP)
-        }))
-        alert.addAction(UIAlertAction(title: String.localized("cancel"), style: .cancel, handler: nil))
-        present(alert, animated: true, completion: nil)
-    }
-
-    private func showKeyManagementDialog(_: Row) {
-        let alert = UIAlertController(title: String.localized("pref_manage_keys"), message: nil, preferredStyle: .safeActionSheet)
-        alert.addAction(UIAlertAction(title: String.localized("pref_managekeys_export_secret_keys"), style: .default, handler: { _ in
-            let msg = String.localizedStringWithFormat(String.localized("pref_managekeys_export_explain"), self.externalPathDescr)
-            let alert = UIAlertController(title: nil, message: msg, preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: String.localized("ok"), style: .default, handler: { _ in
-              self.startImex(what: DC_IMEX_EXPORT_SELF_KEYS)
-            }))
-            alert.addAction(UIAlertAction(title: String.localized("cancel"), style: .cancel, handler: nil))
-            self.present(alert, animated: true, completion: nil)
-        }))
-        alert.addAction(UIAlertAction(title: String.localized("pref_managekeys_import_secret_keys"), style: .default, handler: { _ in
-            let msg = String.localizedStringWithFormat(String.localized("pref_managekeys_import_explain"), self.externalPathDescr)
-            let alert = UIAlertController(title: nil, message: msg, preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: String.localized("ok"), style: .default, handler: { _ in
-                self.startImex(what: DC_IMEX_IMPORT_SELF_KEYS)
-            }))
-            alert.addAction(UIAlertAction(title: String.localized("cancel"), style: .cancel, handler: nil))
-            self.present(alert, animated: true, completion: nil)
+            let documents = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)
```

Comment:
you can use startImex here, reduces code duplication

Reply:
Fixed